lib/pypechain/pypechain/run_pypechain.py

```diff
@@ -20,6 +21,19 @@
 from web3.types import ABIFunction
 
 
+def format_and_write_code(path, code):
```

Comment:
I'd rather this function just be a singular responsibility.  Let's just have it format the code.  Make sure that we are providing types for the parameters and return value and sphynx docs.

Always be weary when you have 'and' in a function name.  sometimes its ok, usually a code smell though.

Leave the writing of the file in the top level for now, its not so bad yet.

Reply:
but.. 2 lines of code could be 1 🫣